<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Folder Path</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="FolderList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="iconSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideRight</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>202</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Folder Path</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Repository Name</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="Browse">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>280</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Delete">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>85</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FolderPath">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>241</width>
     <height>30</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Add">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RepositoryName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>241</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Repository Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Upload">
   <property name="geometry">
    <rect>
     <x>471</x>
     <y>135</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Reset">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>450</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ID">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>21</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_PW">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>21</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>PW</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PW">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>450</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="Set">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>400</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
